airgun/views/subscription.py

```diff
@@ -21,6 +21,16 @@
 )
 
 
+def wait_for_spinner(widget):
```

Comment:
We have pretty similar helper for rhrepo. Maybe it's time to move it out somewhere? I think there might be more new style pages (or more upcoming).

Reply:
Are there others spinners on rhrepo page, can't we just use the same locator?

Reply:
nope, I wanted to replay and find my self repeating what I said above :)